index.html

```diff
@@ -318,7 +319,8 @@ <h1 class="about-header header">About me</h1>
             maxlength="500"
           ></textarea>
         </label>
-        <button type="submit" class="project-btn submit btn">
+        <p id="error"></p>
+        <button type="submit" id="submit" class="project-btn submit btn">
           Get in touch
         </button>
```

Comment:
- [ ] email field containing uppercase characters should generate an error as per the project requirements. You can check if the email is all lowercase by comparing if the email and it's lowercase converted version are the same, ie: `email === email.toLowerCase()`, utilizing regular expressions, or looping through each letter of the email string.
Kindly implement this.

Reply:
" email field containing uppercase characters should generate an error as per the project requirements." This requirement is fulfilled I kindly suggest you check the project on the local server and not the live link as this branch is not merged in the main branch